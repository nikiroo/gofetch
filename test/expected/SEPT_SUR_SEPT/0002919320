   L'EX-STAR DU CYCLISME JAN ULLRICH ENCORE SOUPÇONNÉ D'AGRESSION  

  [Cyclisme] 26-09-2018 17:34

  o Reference: 0002919320
  o News link: http://s.7s7.be/3480000
  o Source link: 


    Par: rédaction
    
    26/09/18 - 17h34 Source: Belga    Jan Ullrich © afp.
    
    L'ex-champion  cycliste  allemand  Jan  Ullrich,  déjà impliqué
    dans  plusieurs  affaires  de  violences ces derniers mois, est
    accusé   d'avoir   agressé  mardi  un  homme  à  l'aéroport  de
    Hambourg, a indiqué mercredi un porte-parole de la police.
    
    Ullrich,  44  ans, aurait "soudainement agressé" mardi un homme
    de  34  ans  dans la zone de contrôle de sécurité de l'aéroport
    de  Hambourg,  selon  ce  porte-parole,  interrogé par l'agence
    SID, filiale de l'AFP.
    
    La  victime  présumée n'ayant porté plainte que quelques heures
    plus  tard,  Ullrich  avait  entretemps  pris un avion pour les
    Etats-Unis.   Il   est  censé  y  suivre,  selon  le  quotidien
    populaire  Bild, un traitement contre ses addictions à l'alcool
    et aux drogues.
    
    Unique  vainqueur allemand du Tour de France, Jan Ullrich avait
    déjà  été  accusé  en  août  d'avoir  agressé une escort-girl à
    Francfort.
    
    Quelques  jours  plus  tôt, Jan Ullrich avait été arrêté par la
    police  en  Espagne,  où il vit, pour s'être introduit de force
    chez   son   voisin,   l'acteur  et  réalisateur  allemand  Til
    Schweiger, et avoir provoqué une bagarre dans son jardin.
    
    Après  sa  garde  à vue, il avait entamé une cure dans l'espoir
    de  revoir ses trois enfants, dont la mère a la garde depuis la
    séparation du couple fin 2017.
    
    Il  avait été alors admis dans un hôpital psychiatrique, où lui
    avait  rendu  visite  l'Américain  Lance  Armstrong,  son grand
    rival à la fin des années 90.
    
    Monstre  de  puissance,  rouleur  hors pair, Jan Ullrich, formé
    dans  l'ex-Allemagne de l'Est, était promis à un immense avenir
    mais  a  été plombé par son inconstance et finalement convaincu
    de dopage.



