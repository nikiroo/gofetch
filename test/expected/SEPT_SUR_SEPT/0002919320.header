0L'ex-star du cyclisme Jan Ullrich encore soupçonné d'agression	null/SEPT_SUR_SEPT/0002919320		70
i  [Cyclisme] 26-09-2018 17:34
i
i    L'ex-champion  cycliste  allemand  Jan  Ullrich,  déjà impliqué
i    dans  plusieurs  affaires  de  violences ces derniers mois, est
i    accusé d'avoir agressé ...
i
