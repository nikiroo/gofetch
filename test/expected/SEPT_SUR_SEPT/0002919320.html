<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>L'ex-star du cyclisme Jan Ullrich encore soupçonné d'agression</h2>
	<div class='details'>([Cyclisme] 26-09-2018 17:34)</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0002919320</a></li>
        <li>News link: <a href='http://s.7s7.be/3480000'>http://s.7s7.be/3480000</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Par: rédaction<br/><br/>26/09/18 - 17h34 Source: Belga    Jan Ullrich © afp.<br/><br/>L&#x27;ex-champion cycliste allemand Jan Ullrich, déjà impliqué dans plusieurs affaires de violences ces derniers mois, est accusé d&#x27;avoir agressé mardi un homme à l&#x27;aéroport de Hambourg, a indiqué mercredi un porte-parole de la police.<br/><br/>Ullrich, 44 ans, aurait &quot;soudainement agressé&quot; mardi un homme de 34 ans dans la zone de contrôle de sécurité de l&#x27;aéroport de Hambourg, selon ce porte-parole, interrogé par l&#x27;agence SID, filiale de l&#x27;AFP.<br/><br/>La victime présumée n&#x27;ayant porté plainte que quelques heures plus tard, Ullrich avait entretemps pris un avion pour les Etats-Unis. Il est censé y suivre, selon le quotidien populaire Bild, un traitement contre ses addictions à l&#x27;alcool et aux drogues.<br/><br/>Unique vainqueur allemand du Tour de France, Jan Ullrich avait déjà été accusé en août d&#x27;avoir agressé une escort-girl à Francfort.<br/><br/>Quelques jours plus tôt, Jan Ullrich avait été arrêté par la police en Espagne, où il vit, pour s&#x27;être introduit de force chez son voisin, l&#x27;acteur et réalisateur allemand Til Schweiger, et avoir provoqué une bagarre dans son jardin.<br/><br/>Après sa garde à vue, il avait entamé une cure dans l&#x27;espoir de revoir ses trois enfants, dont la mère a la garde depuis la séparation du couple fin 2017.<br/><br/>Il avait été alors admis dans un hôpital psychiatrique, où lui avait rendu visite l&#x27;Américain Lance Armstrong, son grand rival à la fin des années 90.<br/><br/>Monstre de puissance, rouleur hors pair, Jan Ullrich, formé dans l&#x27;ex-Allemagne de l&#x27;Est, était promis à un immense avenir mais a été plombé par son inconstance et finalement convaincu de dopage.
	</div>
<hr/>
</div>
</body>
